symfony/console/Symfony/Component/Console/Tests/ApplicationTest.php

```diff
@@ -45,6 +45,8 @@ public static function setUpBeforeClass()
         require_once self::$fixturesPath.'/Foo5Command.php';
```

Comment:
Can you please drop those unneeded tests and put these folders into the `.gitignore`? Thanks :)